src/System.Net.HttpListener/src/System/Net/Windows/HttpListener.Windows.cs

```diff
@@ -1915,7 +1915,7 @@ internal unsafe DisconnectAsyncResult(HttpListener httpListener, ulong connectio
                 _httpListener = httpListener;
                 _connectionId = connectionId;
                 // we can call the Unsafe API here, we won't ever call user code
-                _nativeOverlapped = httpListener._requestQueueBoundHandle.AllocateNativeOverlapped(s_IOCallback, state: this, pinData: null);
+                _nativeOverlapped = httpListener.RequestQueueBoundHandle.AllocateNativeOverlapped(s_IOCallback, state: this, pinData: null);
```

Comment:
Simple change. I tried this fix when submitting the tests, but found things still didn't work